browser/tor/brave_tor_browsertest.cc

```diff
@@ -394,6 +422,27 @@ IN_PROC_BROWSER_TEST_F(BraveTorTestWithCustomProfile, Incognito) {
   EXPECT_TRUE(is_element_enabled("torSnowflake"));
 }
 
+IN_PROC_BROWSER_TEST_F(BraveTorTestWithCustomProfile, Autofill) {
+  GURL fake_url("http://brave.com/");
+  // Disable autofill in private windows.
```

Comment:
this says "disable in private windows", but then you only test a Tor window. You also need to test a regular incognito window